packages/core/src/Utils/Datadog/index.ts

```diff
@@ -43,7 +51,23 @@ if (isProduction || isStaging) {
         version: dataDogVersion,
         trackFrustrations: true,
         enableExperimentalFeatures: ['clickmap'],
-        excludedActivityUrls: [/^https:\/\/api.telegram.org.*$/],
+        beforeSend: event => {
+            if (event.type === 'resource') {
+                event.resource.url = event.resource.url.replace(
+                    /^https:\/\/api.telegram.org.*$/,
+                    'telegram token=REDACTED'
+                );
+
+                if (event.resource.url.match(/^ https:\/\/eu.deriv.com\/\/ctrader-login.*$/)) {
+                    const url = event.resource.url;
+                    const accnt = getAcct1Value(url);
+                    event.resource.url = event.resource.url.replace(
+                        /^ https:\/\/eu.deriv.com\/\/ctrader-login.*$/,
```

Comment:
```suggestion
                        /^ https:\/\/eu.deriv.com\/ctrader-login.*$/,
```
I think `\/\/` is for `//` in `https://` but for `/ctrader` we should have just one of it. right?